lib/driver.rb

```diff
@@ -0,0 +1,109 @@
+require_relative 'csv_record'
+require 'csv'
+require 'time'
+require 'awesome_print'
+
+module RideShare
+  class Driver < CsvRecord
+    attr_reader :id, :name, :vin, :trips
+    attr_accessor :status #Wave 3: overriding status in dispatch, need to make this attr accessor
+    
+    def initialize(
+      id:,
+      name:,
+      vin:,
+      status: :AVAILABLE,
+      trips: nil
+      )
```

Comment:
If you indent your argument list it makes it easier to distinguish from the method body.

```suggestion
    def initialize(
        id:,
        name:,
        vin:,
        status: :AVAILABLE,
        trips: nil
      )
```